src/js/bellows.js

```diff
@@ -52,6 +52,12 @@
             this._bindEvents();
         },
 
+        destroy: function() {
+            this.$bellows
+                .removeData(this.name)
+                .off(this.options.event);
```

Comment:
Why is the event in the default variable? Should we just take it out of the default variable and be not override-able like how we do it for all of our other plugins?